packages/inula-adapter/examples/$nextTick/nextTick.jsx

```
@@ -0,0 +1,18 @@
+export default function (props) {
+  // inula rendering is synchronous, so this timeout appends this function at next asynchronous position to execute after render
+  useEffect(() => {
+    // this timeout should be wrapped as an adapter that way you can call only $nextClick( ... ) as usual
+    const timeout = setTimeout(() => {
+      // this must be wrapped, because at the time of render, this element does not exist yet
+      useInstance().$refs.inp.$el.attachEventListener('keyUp', e => {
```

ESLint

[error] 'useInstance' is not defined.

(no-undef)